app/models/audit_results.rb

```diff
@@ -92,16 +81,14 @@ class AuditResults
     CM_STATUS_CHANGED
   ].freeze
 
-  attr_reader :result_array, :druid, :storage_area, :logger
-  attr_accessor :actual_version, :check_name
+  attr_reader :druid, :storage_area, :actual_version, :check_name
 
-  def initialize(druid, actual_version, storage_area, check_name = nil, logger: nil)
+  def initialize(druid:, actual_version:, storage_area:, check_name: nil)
```

Comment:
Appreciate moving to keyword args over positional ones!

Reply:
Maybe I'm being slow, but ... who are the callers and are these changes made there?

Reply:
You're not being slow; this isn't done yet (AFAICT). I don't think this PR has a passing test suite yet. My vague recollection is J-Litt wanted some early feedback on decisions made so far before proceeding further.

Reply:
Correct. I didn't want to change all of the callers and tests until there was consensus on the refactor/split.